PipelinesTasks/PRMetrics/buildAndReleaseTask/invokers/azureReposInvoker.ts

```diff
@@ -0,0 +1,219 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Could you lint this file?

Reply:
It's linted. Does it show eslint errors on your local?

Reply:
This file is showing semicolons and " strings, which are different than the linting standard we have. The test file looks fine in comparison

Reply:
Linted now. Had to move the eslint config file to the base of the task.

Reply:
This will break the build pipe. Instead "npm install" in the root folder and it should work fine for you